src/desdemona/catalogs/sample_catalog.clj

```diff
@@ -1,5 +1,4 @@
-(ns desdemona.catalogs.sample-catalog
-  (:require [desdemona.functions.sample-functions :refer [transform-segment-shape prepare-rows]]))
+(ns desdemona.catalogs.sample-catalog)
```

Comment:
I'm somewhat surprised this works; I thought that _something_ needs to actually require the namespace in which the fn definition lives for it to pop into existence so that onyx can later refer to it...